lab_1/main.py

```diff
@@ -105,7 +254,13 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+
+    with open(path_to_file, 'w') as report:
+        for text in content:
+            context = ' '.join(text) + '\n'
+            report.write(context)
```

Comment:
first, put each context in one variable
only then write it to file
it's not good to write to file at each step